Engine/src/Input/AxisCode.bf

```diff
@@ -0,0 +1,28 @@
+namespace SteelEngine.Input
+{
+	public enum AxisCode
+	{
+		MouseX = 0,
+		MouseY,
+		MouseScroll,
+
+		GamepadLeftStickX,
+		GamepadLeftStickY,
+		GamepadRightStickX,
+		GamepadRightStickY,
+		GamepadLeftTrigger,
+		GamepadRightTrigger,
```

Comment:
Is the Gamepad prefix necessary here? Be nice to have this somewhat shortened if we never anticipate a conflict in this enum.